impl/src/main/java/com/sun/faces/context/IllegalXmlCharacterFilterWriter.java

```diff
@@ -0,0 +1,56 @@
+package com.sun.faces.context;
+
+ import java.io.FilterWriter;
+import java.io.IOException;
+import java.io.Writer;
+
+ public class IllegalXmlCharacterFilterWriter extends FilterWriter {
```

Comment:
As suggested in https://github.com/eclipse-ee4j/mojarra/pull/4517#issuecomment-441930544 I would highly recommend to consider the performance considerations implemented in https://github.com/apache/myfaces/blob/master/impl/src/main/java/org/apache/myfaces/util/IllegalXmlCharacterFilterWriter.java, see https://issues.apache.org/jira/browse/MYFACES-4266 for the reasons.